tests/unit-core/test-to-property-descriptor.c

```diff
@@ -139,14 +151,18 @@ main (void)
   source_p = "({ set: function(v) {}, enumerable:true, configurable:false })";
   object = create_property_descriptor (source_p);
 
-  to_property_descriptor (object, &prop_desc);
+  prop_desc = to_property_descriptor (object);
 
   check_attribute (prop_desc.setter, object, "set");
 
-  TEST_ASSERT (!prop_desc.is_value_defined && !prop_desc.is_writable_defined);
-  TEST_ASSERT (!prop_desc.is_get_defined && prop_desc.is_set_defined);
-  TEST_ASSERT (prop_desc.is_enumerable_defined && prop_desc.is_enumerable);
-  TEST_ASSERT (prop_desc.is_configurable_defined && !prop_desc.is_configurable);
+  TEST_ASSERT ((prop_desc.flags & JERRY_PROP_IS_VALUE_DEFINED) == 0);
+  TEST_ASSERT ((prop_desc.flags & JERRY_PROP_IS_WRITABLE_DEFINED) == 0);
+  TEST_ASSERT ((prop_desc.flags & JERRY_PROP_IS_GET_DEFINED) == 0);
+  TEST_ASSERT (prop_desc.flags & JERRY_PROP_IS_SET_DEFINED);
+  TEST_ASSERT (prop_desc.flags & JERRY_PROP_IS_ENUMERABLE_DEFINED);
+  TEST_ASSERT (prop_desc.flags & JERRY_PROP_IS_ENUMERABLE);
+  TEST_ASSERT (prop_desc.flags & JERRY_PROP_IS_CONFIGURABLE_DEFINED);
+  TEST_ASSERT ((prop_desc.flags & JERRY_PROP_IS_CONFIGURABLE) == 0);
```

Comment:
I would use `!(prop_desc.flags & JERRY_PROP_IS_CONFIGURABLE)` since in the treu cases you don't use `!= 0` checks. Same for all checks above.